src/api/supabase/createClient.ts

```diff
@@ -0,0 +1,20 @@
+import { createClient } from '@supabase/supabase-js';
```

Comment:
make sure to delete this file as the createClient file already exists in the `api/supabase` route